src/main/scala/com/cognite/sdk/scala/v1/transformations.scala

```diff
@@ -78,10 +78,12 @@ sealed case class RawDataSource(`type`: String, database: String, table: String)
 sealed case class SequenceRowDataSource(`type`: String, externalId: String)
     extends DestinationDataSource
 
+final case class TransformationRun(externalId: String)
+
 final case class JobDetails(
-    id: Int,
+    id: Long,
     uuid: String,
-    transformationId: Int,
+    transformationId: Long,
```

Comment:
this also

Reply:
same comment as https://github.com/cognitedata/cognite-sdk-scala/pull/568#discussion_r994310431